cwms-data-api/src/main/webapp/regexp.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
```

Comment:
Great addition.

Reply:
If users need help figuring out how to write something tricky in Oracle syntax we can add to these examples.

Reply:
Actually, maybe we should say that - Request help from HEC if you would like help crafting a regular expression or have suggested examples to add to this doc.  I'm gonna merge this now and let you figure out how to add to or fix that part

Reply:
I would word that as "As for help from HEC or open a new discussion at <link to github discussion>" but yes that's a good idea.